src/__experimental__/components/grouped-character/grouped-character.component.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Textbox from '../textbox';
+import { generateGroups, toSum, toIndexSteps } from './grouped-character.utils';
+import Events from '../../../utils/helpers/events/events';
```

Comment:
There is a `package.json` so we don't need to use specific file
we can just use `'../../../utils/helpers/events`

Reply:
changed